package.json

```diff
@@ -26,25 +26,29 @@
     "@types/react-redux": "^7.1.7",
     "@types/react-router-dom": "^5.1.3",
     "@types/webpack-env": "^1.15.1",
+    "@types/recharts": "^1.8.10",
     "axios": "^0.19.2",
     "clsx": "^1.1.0",
     "moment": "^2.26.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-redux": "^7.2.0",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.4.0",
+    "react-scripts": "3.4.1",
+    "recharts": "^1.8.5",
     "typescript": "^3.7.5"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "CI=true react-scripts start",

```

Comment:
What does the `CI=true` do?

Reply:
Not exactly sure what it does but it fixes the issue where it fails to start in the docker container.
https://github.com/facebook/create-react-app/issues/8688#issuecomment-602084087